services/webnn/ort/graph_builder_ort.h

```diff
@@ -94,25 +94,26 @@ class GraphBuilderOrt {
   const mojom::Operand& GetOperand(uint64_t operand_id);
   std::string GetOperandName(uint64_t operand_id);
 
-  // Some initializers must be uploaded to raw data, e.g. Reshape op needs
-  // parameter *shape* as raw data to do shape inference.
+  // Some initializers must be uploaded to raw data, for example:
+  // 1. Reshape op needs parameter *shape* as raw data to do shape inference.
+  // 2. Reduce op needs parameter *axes* as raw data.
```

Comment:
How can we know which op's input operand should be initialized as raw data?

Reply:
Because runtime error message told me external data for this op's input is invalid.